README.md

```
@@ -0,0 +1,180 @@
+import httpx
+
+response = httpx.post("http://localhost:8000/api/v1/query", json={
+    "query": "How do I configure authentication in FastAPI?",
+    "context": "web_development"
+})
+
+print(response.json())
+```
+
+### Document Ingestion
+```python
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)